pkg/apis/extensions/validation/operatingsystemconfig.go

```diff
@@ -197,6 +207,35 @@ func ValidateContainerdRegistryConfigs(registries []extensionsv1alpha1.RegistryC
 	return allErrs
 }
 
+var availablePluginPathOperations = sets.New(extensionsv1alpha1.AddPluginPathOperation, extensionsv1alpha1.RemovePluginPathOperation)
+
+func validateContainerdPluginConfigs(config *extensionsv1alpha1.ContainerdConfig, fldPath *field.Path) field.ErrorList {
```

Comment:
Does it make sense to add validation that forbids setting `values` when `op: remove`?